core/cmd/chain_commands.go

```diff
@@ -108,3 +114,78 @@ func (cli *Client) RemoveChain(c *cli.Context) (err error) {
 	fmt.Printf("Chain %v deleted\n", c.Args().First())
 	return nil
 }
+
+func (cli *Client) ConfigureChain(c *cli.Context) (err error) {
+	chainID := c.Int64("id")
+	if chainID == 0 {
+		return cli.errorOut(errors.New("missing chain ID [-id integer]"))
+	}
+
+	if !c.Args().Present() {
+		return cli.errorOut(errors.New("must pass in the chain's parameters [-id integer] [JSON blob | JSON filepath]"))
```

Comment:
Does it clobber if you omit keys? For example:

`{"foo": 1}` will erase `{"bar": 2}` or leave it be? Does setting explicit nulls clobber?

Reply:
It allows partial updates, `foo=1` will set `foo` but ignore `bar`. `foo=1 bar=nil` will clear `bar`.